src/test/a11y/tests/admin-routes.test.ts

```diff
@@ -84,4 +84,42 @@ describe('Accessibility - Admin Routes', () => {
             testAccessibility(route.path, route.parameter, route.postMethod, route.postBody);
         });
     });
+
+    describe('Page with Errors', () => {
+        describe('Create Admin Account Page', () => {
+            const url = '/create-admin-account';
+
+            describe('with no input data', () => {
+                testAccessibility(url, '', true, { 'input-autocomplete': '' });
```

Comment:
'input-autocomplete' is not the right post body for this test. Please check the controller code